### ISSUE TITLE
Warning since 6.4.1: Parameter 'Recipients' needs to be a hash with data

### ISSUE THREAD

**Issue by the-me1988** [znuny/Znuny#270]
### Environment

- OS:           
- Browser:      
- Znuny version:  6.4.1

### Expected behaviour

If I answer a ticket, no error message should be thrown.

### Actual behaviour

An error is triggered: 

```
ERROR: OTRS-otrs.Console.pl-Maint::PostMaster::MailAccountFetch-10 Perl: 5.32.1 OS: linux Time: Thu Jul 28 15:16:21 2022

 Message: Parameter 'Recipients' needs to be a hash with data!

 Traceback (432232):
   Module: Kernel::System::Mention::SendNotification Line: 97
   Module: Kernel::System::Ticket::Event::Mentions::Run Line: 68
   Module: Kernel::System::EventHandler::EventHandler Line: 224
   Module: Kernel::System::EventHandler::EventHandlerTransaction Line: 264
   Module: Kernel::System::ObjectManager::ObjectEventsHandle Line: 426
   Module: Kernel::System::ObjectManager::ObjectsDiscard Line: 474
   Module: Kernel::System::ObjectManager::DESTROY Line: 643
   Module: (eval) Line: 193
   Module: main::Start Line: 193
   Module: /usr/share/otrs/bin/otrs.Daemon.pl Line: 152
```

### How to reproduce

It always appears on answering to tickets

### Additional information

Since the update from Znuny 6.3.4 to 6.4.1 I get an error in the apache error log and in the SchedulerTaskWorkerERR.log file:

It seems to work correctly: Sending and recieving mails works like before

**Comment by hanneshal** [comment]
@the-me1988 thanks for the report. I modified it to respect our template. Please check that I don't missed anything

**Comment by hanneshal** [comment]
@the-me1988 if you disable the notification for "mentions", is the warning gone?

**Comment by the-me1988** [comment]
Ah I see a new event method is created: I have set it to "invalid", but the message still appears

**Comment by hanneshal** [comment]
Ok, we found it. 
A fix is on the way.

**Comment by hanneshal** [comment]
https://github.com/znuny/Znuny/commit/142c9f4bf7b59f286af517af487cbe73246fd104

**Comment by hanneshal** [comment]
Fixed with 6.4.2
